02-peer-protocol.md

```diff
@@ -197,6 +200,19 @@ know this node will accept `funding_satoshis` greater than or equal to 2^24.
 Since it's broadcast in the `node_announcement` message other nodes can use it to identify peers 
 willing to accept large channel even before exchanging the `init` message with them. 
 
+#### Defined Channel Types
+
+Channel types are an explicit enumeration: for convenience of future
+definitions they reuse even feature bits, but they are not an
+arbitrary combination (they represent the persistent features which
+affect the channel operation).
+
+The currently defined types are:
+  - no features (no bits set)
```

Comment:
From my reading of the behavior here, you _must_ set a bit, as otherwise, which channel type is used? Seems that would fall back to the implicit negotiation, which is what we're trying to avoid here in the first place...

Reply:
Nothing set is simply the LN 1.0 channel type (before `static_remotekey`)